docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/matrix/inverse-14.rst

```diff
@@ -93,13 +89,7 @@ Algorithm pseudocode:
    * Solve linear equation :math:`Ux = y` for x (backward substitution)
    * Set x as the corresponding column of the output inverse matrix :math:`A^{-1}`
 
-5. If adjoint == true, then it is necessary to multiply :math:`A^{-1}` by its determinant.
-
-   * As explained above, it is enough to compute :math:`det(U)`, since :math:`det(U) = det(A)`.
-   * :math:`det(U)` is just a multiplication of its diagonal elements.
-   * Account for each row swap in the LU decomposition step - for every row swap, swap the sign of the determinant.
-   * Multiply all elements of :math:`A^{-1}` by the determinant to obtain the adjugate matrix.
-
+5. If adjoint == true, then it is sufficient to tranpose the resulting matrix.
```

Comment:
this step should be the first one before computing inverse

Reply:
Fixed!